Marlin/src/gcode/feature/fwretract/M207-M209.cpp

```diff
@@ -63,11 +63,13 @@ void GcodeSuite::M208() {
  *   moves will be classified as retraction.
  */
 void GcodeSuite::M209() {
-  if (MIN_AUTORETRACT <= MAX_AUTORETRACT) {
-    if (parser.seen('S')) {
-      fwretract.enable_autoretract(parser.value_bool());
+  #if ENABLED(FWRETRACT_AUTORETRACT)
+    if (MIN_AUTORETRACT <= MAX_AUTORETRACT) {
+      if (parser.seen('S')) {
+        fwretract.enable_autoretract(parser.value_bool());
+      }
     }
-  }
+  #endif
```

Comment:
It would be better to disable M209 completely, so the G-code throws an “unknown G-code” error when this feature is not enabled.